host/tests/configs/example_rack_config.yml

```diff
@@ -65,52 +56,30 @@ rackConfig:
                 # [ type: "ssh": port: 22 username: "test" password: "test" ]
                 # [ type: "telnet": port: 23 username: "test" password: "test" ]
                 - dut:
-                    ip: "127.0.0.1"  # IP Address of the ADA Hub
-                    description: "local PC"
-                    platform: "linux PC"
+                    ip: "127.0.0.1"
+                    description: "element"
```

Comment:
while pushing this, make the platform platform as TV

Reply:
updated with tv